applications/web_app/fullstack/zeroday-fullstack/template/frontend/src/components/navigation/NavigationMain.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+import { Dropdown, Menu } from 'semantic-ui-react';
+import * as PropTypes from 'prop-types';
+
+import { logout } from '../../actions/auth';
+import { userPropType } from '../../types/customPropTypes';
+import LanguageSwitcher from '../translation/LanguageSwitcher';
+import Translatable from '../translation/Translatable';
+
+const handleHome = (history) => history.push('/');
+const handleProfile = (history) => history.push('/profile');
+const handleLogout = (dispatch) => dispatch(logout());
+
+const NavigationMain = (props) => {
+  const { user, history, dispatch } = props;
+
+  return (
+    <Menu borderless fixed="top">
+      <Menu.Item className="header" onClick={() => handleHome(history)}>
+        <span className="logo" />
+        <img alt="" className="navbar-logo" src="images/eficode_logo_black_200x150.png" />
```

Comment:
@mikkilevon @venetastefanova What about template branding? Do we want Eficode logos on the template example UI?

Reply:
So the Eficode stuff should be replaced with some ACME CORP Logos?
@mikkilevon @zeeket @jpvg10

Reply:
If the customer is going to replace it anyways, does it matter if we use ACME CORP or Eficode as the placeholder? It seems like most "generic company logo" material on the internet has a commercial use license that costs a little bit of money. Ideally the placeholder would look good enough to be presentable in demos.